packages/rancher-istio/charts/templates/istio-install-job.yaml

```diff
@@ -51,6 +56,9 @@ spec:
           configMap:
             name: istio-installer-overlay
         {{- end }}
+{{- if and .Values.installer.releaseMirror.additionalIstioImage.enabled .Values.installer.releaseMirror.additionalIstioImage.volumes }}
```

Comment:
I think we do not need to check if `.Values.installer.releaseMirror.additionalIstioImage.volumes` exists,  it must exist since it is predefined in the value.yaml file.

In line 45, we do not check it.